examples/international_address_autocomplete.js

```diff
@@ -2,38 +2,44 @@ const SmartySDK = require("smartystreets-javascript-sdk");
 const SmartyCore = SmartySDK.core;
 const Lookup = SmartySDK.internationalAddressAutocomplete.Lookup;
 
-// US Autocomplete Pro only supports using Embedded Keys
-let key = process.env.SMARTY_EMBEDDED_KEY;
+// International Autocomplete only supports using Embedded Keys
```

Comment:
In the docs I saw that it does support secret keys, but that it is not recommended to do so?

Reply:
Oh let me check! I must have missed that

Reply:
The API does support secret keys. I'll add in the snippet for that!